### ISSUE TITLE
Add information about how long compression took

### ISSUE THREAD

**Issue by fstanis** [GoogleChromeLabs/squoosh#291]
Maybe put number of milliseconds the operation took right under compressed image size?

Would be nice for (rough) comparison of WASM performance across browsers / devices.

**Comment by kosamari** [comment]
Hi @fstanis, hmm but wouldn't the result vary depending on the machine?

I don't think target use case for Squoosh is WASM performance testing, and if someone need to get those data from squoosh, they'd probably better off with browser trace?

**Comment by fstanis** [comment]
I agree it wouldn't work well for any serious performance testing, but I'm still often curious how much time it takes to do e.g. quant vs mozjpeg.

I don't think it's that important for the project, but might be a good first issue and some users might find it interesting, that's all :)

**Comment by jakearchibald** [comment]
The hard part here is UI. On mobile we're already overflowing the results data.
